R-proj/src/rotating.cpp

```diff
@@ -86,13 +90,14 @@ Rcpp::NumericMatrix rotating (Rcpp::Reference P, Rcpp::Nullable<Rcpp::NumericMat
                 TransorfMat = Rcpp::as<MT>(T);
                 ZP.linear_transformIt(TransorfMat.inverse());
             } else {
-                TransorfMat = rotating < MT > (ZP);
+                TransorfMat = rotating < MT > (ZP, seed2);
             }
             Mat = extractMatPoly(ZP);
             break;
         }
         case 4: {
-            Vpolytope VP1;
+            throw Rcpp::exception("volesti does not support rotation for this representation currently.");
+            /*Vpolytope VP1;
```

Comment:
is this piece of code going to be used in the future?

Reply:
I removed this part.